src/LogViewer/chunk-worker.js

```diff
@@ -128,14 +128,21 @@ const update = (response) => {
 const error = () => self.postMessage(JSON.stringify({ type: 'error' }));
 const loadEnd = () => self.postMessage(JSON.stringify({ type: 'loadend' }));
 
-const toHtml = (index) => {
+const getParagraphClass = (lineNumber, metadata) => {
+  const { highlightStart, highlightEnd } = metadata;
+
+  return lineNumber >= highlightStart && lineNumber <= highlightEnd ? 'highlight' : '';
```

Comment:
Let's break this line up at the ternary parts so it is a little easier to read and understand what's going on:

```js
return lineNumber >= highlightStart && lineNumber <= highlightEnd ?
  'highlight' :
  '';
```